### PR TITLE
malware readme and desc updates

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [x] Ready

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-3010

## Minimum version of Cortex XSOAR
- [x] 6.0.0

## Does it break backward compatibility?
   - [x] No

### CODE REVIEW

**Comment by bakatzir** [comment]
failed only on 6.2 https://code.pan.run/xsoar/content/-/pipelines/3016908, IDK why @yaakovi.
the pack should be fromversion 6.5.0, per the PRD.

**Comment by bakatzir** [comment]
@yaakovi please also check that I did the relative file path correctly.
I used https://github.com/demisto/content/edit/master/Packs/GoogleCalendar/Integrations/GoogleCalendar/README.md as reference.

File: Packs/MalwareInvestigationAndResponse/README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Tagging of malicious and benign indicators to tie into allow & deny lists for th
 Incident mirroring & closure with EDR
 Support for aggregate incidents from EDRs, which may include multiple alerts & files
 
-![Malware Investigation & Response Incident Handler](https://raw.githubusercontent.com/demisto/content/78e4fb246c3fa57e6201ee2791fd4919864ed956/Packs/MalwareInvestigationAndResponse/doc_files/Malware_Investigation_%26_Response_Incident_Handler.png)
+![Malware Investigation & Response Incident layout](./../../doc_files/Malware_Investigation_&_Response_layout.png)
```
Comment (yaakovi) [inline Packs/MalwareInvestigationAndResponse/README.md:24]:
```suggestion
![Malware Investigation & Response Incident layout](./doc_files/Malware_Investigation_&_Response_layout.png)
```

**Review by yaakovi — APPROVED** [review]

File: Packs/MalwareInvestigationAndResponse/README.md (hunk 1 above)
Comment (bakatzir) [reply Packs/MalwareInvestigationAndResponse/README.md]: Relative URLs are not supported in the pack readme, used absolute URLs instead.
https://app.circleci.com/pipelines/github/demisto/content/167335/workflows/fa548c75-63eb-4333-ae8c-f83c035bfd71/jobs/525463

**Comment by bakatzir** [comment]
@ssokolovich @altmannyarden ^^

**Comment by ssokolovich** [comment]
@bakatzir - question from my side, is the pack name change can affect other packs that have a dependency on it? 
Or for example, fail future builds?

**Comment by ShahafBenYakir** [comment]
@bakatzir yes that is true

**Pull request merged**